src/ui/buttons/buttons.jsx

```diff
@@ -242,6 +266,20 @@ export function Buttons(props: ButtonsProps): ElementNode {
     return i;
   };
 
+  const showTagline =
+    tagline &&
+    layout === BUTTON_LAYOUT.HORIZONTAL &&
+    !fundingSource &&
+    !message;
+  const showPoweredBy =
+    layout === BUTTON_LAYOUT.VERTICAL && fundingSources.includes(FUNDING.CARD);
```

Comment:
I think `fundingSources` will always include `FUNDING.CARD` - https://github.com/paypal/paypal-checkout-components/blob/fbe73816fafc65d6fde81bb43d4107548e6effd7/src/ui/buttons/buttons.jsx#L240-L245 (so can remove the `&& fundingSources.includes`)

Reply:
I actually don't think it would. Its confusing, but the standalone button case uses `Buttons` component but with `fundingSource=paypal|venmo|card|etc`. I think this check is precisely for those instances where we are in Standalone and its not the Card button

Reply:
ah good catch! thank you for clarifying that!

Reply:
Not a problem, actually that would be a good candidate for a refactor. It would probably simplify things if we created a `StandaloneButton` component and rendered that instead of `Buttons` if `fundingSource` was passed